<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="THUMTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Save Thumnail</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0" colspan="2">
         <spacer name="THUMSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="THUMNameStringLabel">
             <property name="text">
              <string>Name Text:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QExtendedLineEdit" name="THUMNameString"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="THUMLevelLabel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Level:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QExtendedLineEdit" name="THUMLevel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="FLAGTab">
       <attribute name="title">
        <string>Flag</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="GAMETab">
       <attribute name="title">
        <string>Game</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="TIMETab">
       <attribute name="title">
        <string>Time</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="PCPMTab">
       <attribute name="title">
        <string>Position</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="CAMDTab">
       <attribute name="title">
        <string>Camera</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ITEMTab">
       <attribute name="title">
        <string>Items</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="WTHRTab">
       <attribute name="title">
        <string>Weather</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="MINETab">
       <attribute name="title">
        <string>Ether Mines</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="TBOXTab">
       <attribute name="title">
        <string>Treasure Box</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="OPTDTab">
       <attribute name="title">
        <string>Options</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QExtendedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>QExtendedWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
